.travis.yml

```diff
@@ -18,7 +18,7 @@ matrix:
   include:
     - php: 5.5
       env:
-        - COMPOSER_FLAGS="--prefer-lowest --prefer-dist --no-interaction"
+        - COMPOSER_FLAGS="--prefer-lowest --prefer-dist --no-interaction -vvv"
```

Comment:
this seems the only way that travis end up with 10m without output error 🙈

Reply:
dont like it 🙊 @danrot what do you think?

Reply:
@wachterjohannes I also don't like it but its the only way composer does a output and travis does not say there is no 10m output